script/script.go

```diff
@@ -25,18 +26,48 @@ import (
 	gjson "layeh.com/gopher-json"
 )
 
-// InitialPoolSize is the number of lua states created initially per route
-var InitialPoolSize int = 3
+const (
+	defaultPoolSize int = 10
+)
+
+type luaScript struct {
+	// PoolSize is the number of lua states stored per route - there may be more parallel
+	// requests, but only this number is cached.
+	PoolSize int
 
-// MaxPoolSize is the number of lua states stored per route - there may be more parallel
-// requests, but only this number is cached.
-var MaxPoolSize int = 10
+	// number of filter instances
+	instances int
```

Comment:
what do we use this field for? As I see, we're only increment it currently. Let's remove it